viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -7,6 +7,7 @@
 #include <grpcpp/server_context.h>
 
 #include <atomic>
+#include <ctime>
```

Comment:
why is this needed?

Reply:
We now are calling std::time on line 660 of slam_service.cc. however, since we import file_handler.h which also imports this